docs/design/storage-auto-scale.md

```diff
@@ -32,13 +32,17 @@ Implement auto storage scale CR per device class of a specific Storagecluster.
         phase: (NotStarted | InProgress | Succeeded | Failed)
         error:
             message: ""
-        lastExpansionStartTime: (always set after a new expansion is started)
-        lastExpansionCompletionTime: (only set on success)
```

Comment:
@parth-gr shouldn't we also set it on `Failed` so the user knows when the failure occured?